core/vm/lightclient/types.go

```diff
@@ -209,15 +209,19 @@ func DecodeHeader(input []byte) (*Header, error) {
 	return &header, nil
 }
 
+type KeyChecker func(string) bool
```

Comment:
Suggest `KeyChecker` return error, because it is a bit confusing what is the meaning of a return `true`, is it valid or invalid..